README.md

```diff
@@ -23,23 +23,26 @@
     python manage.py test
 - Run from main app folder in vagrant box where <i>manage.py</i> is present.
 - This will run both the unit tests as well as the PEP 8 tests.
+
+
+#### To run tests for a single app
+    python manage.py test *app_name*
+    
+- Can be run on: 'api', 'pep8' and 'blacklist_manager'
```

Comment:
Does pep8 run on all apps if you select it or is it in here by accident?

If it's in here on purpose, you should list it below... though given it's not an "app" it seems really weird to have it as an option.